### ISSUE TITLE
dax studio not connecting to power bi

### ISSUE THREAD

**Issue by ahdatabase** [DaxStudio/DaxStudio#72]
version:  2.7.4
log data:
2018-05-13 12:01:42.404 +01:00 [Error] "WebRequestFactory" "RequiresProxyCredentials" "Unable to connect to the remote server"
2018-05-13 12:01:42.438 +01:00 [Error] "WebRequestFactory" "InitializeAsync" "call to GetProxy failed"
2018-05-13 12:01:42.448 +01:00 [Error] "DaxFormatter" "PrimeConnectionAsync" "Unable to connect to the remote server"
2018-05-13 12:01:44.122 +01:00 [Error] "WebRequestFactory" "RequiresProxyCredentials" "Unable to connect to the remote server"
2018-05-13 12:01:44.122 +01:00 [Error] "WebRequestFactory" "InitializeAsync" "call to GetProxy failed"
2018-05-13 12:01:44.126 +01:00 [Error] "VersionCheck" "ServerVersion.get" "Unable to connect to the remote server"
2018-05-13 12:01:44.137 +01:00 [Error] "VersionCheck" "ServerVersion.get" "Unable to connect to the remote server"
2018-05-13 12:01:57.157 +01:00 [Error] "VersionCheck" "ServerVersion.get" "Unable to connect to the remote server"

**Comment by dgosbell** [comment]
None of those log messages relate to connecting to Power BI Desktop. They just related to issues getting through your corporate http proxy while trying to check if a newer version of DAX Studio is available. Can you describe what you were trying to do an what results you observed? Did you have Power BI Desktop running with the file you wanted to connect to open?

**Comment by jm98** [comment]
I see this too in 2.7.4
Have PowerBI desktop open with a pbix file loaded (live query to SSAS Tabular)
When I start DAX Studio, the "Connect" dialog only gives the option to connect to "Tabular Server"

**Comment by dgosbell** [comment]
@jm98 - in the case of a .pbix file with a live query to SSAS Tabular this is the expected behaviour. You should just connect DAX Studio to the model on the SSAS Tabular server.

This is because when we try to connect to the pbix file it contains no data model and no data. And there are currently no supported APIs for us to tell the difference between an empty pbix file and a pbix file with a live query to SSAS.

@ahdatabase - are you also using a .pbix file with a live query to SSAS?

**Comment by jm98** [comment]
@dgosbell  That makes sense.  Just cut out the middle man.  Thanks

**Comment by suprak** [comment]
Something similar is happening for me too in 2.7.4.

My pbix contains a direct query to a SQL db.
I've had DAX studio connect to my model successfully, once or twice. But currently in a state where (even with logging enabled), it briefly attempts to connect but nothing happens.

No idea how to debug this (already enabled logging in the config, nothing is getting logged).

**Comment by dgosbell** [comment]
@suprak Sorry for the delay in responding, I've been having some hardware issues, but I'm back up and running now. 

Your issue is not the same as the first issue in this thread, models using Direct Query should work fine (as opposed to a live connect to SSAS where the model is not part of the PBIX file). Unfortunately I can't reproduce this issue. I've created a simple Direct Query model against the Adventure Works sample database it is working fine. Do you have any none alpha-numeric characters in any of your table or column names or anything unusual like that in your model?

**Comment by agarwalharsh73** [comment]
I am also having the same trouble where I am able to connect to the PBI source only once but later on it tries to establish a connection and then returns back to the status not connected.

**Comment by dgosbell** [comment]
We have not been able to re-produce these issues, but we have released v2.8.0 which has updated data providers. So I'm going to close this issue. If you still experience this problem using v2.8.0 please open a new issue.
